hazelcast/util.py

```diff
@@ -281,3 +281,18 @@ def next(self):
             return None
         idx = random.randrange(0, len(members))
         return members[idx]
+
+
+class IterationType:
+    """To differentiate users selection on result collection on map-wide
+    operations like values, keySet, query etc.
```

Comment:
nit: does it make sense to change the list of operations to `entry_set, key_set, values` to make things more Python client specific?

Reply:
Good idea, changed that